jar-service/src/main/java/com/sixsq/slipstream/ui/UIPlacementResource.java

```diff
@@ -0,0 +1,70 @@
+package com.sixsq.slipstream.ui;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+import com.sixsq.slipstream.resource.BaseResource;
+import org.restlet.data.MediaType;
+import org.restlet.data.Status;
+import org.restlet.representation.Representation;
+import org.restlet.representation.StringRepresentation;
+import org.restlet.resource.Get;
+import org.restlet.resource.Put;
+import org.restlet.resource.ResourceException;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+
+/**
+ *
+ */
+public class UIPlacementResource extends BaseResource {
+
+    private static Logger logger = Logger.getLogger(UIPlacementResource.class.getName());
+
+    @Get("json")
+    public Representation getUI() {
```

Comment:
What "service" do you mean here?  Is it PRS?  Are you planning to call PRS directly from within the `GET`?  Because `/ui/placement` is a resource on SS service (SSCLJ later) and the current `GET` call is just a check for the resource availability, which I guess will not be used on production.  Do we need it then?